### PR TITLE
[Windows] Move Android and VS helpers to dedicated files

### PR DESCRIPTION
**Pull request by vpolikarpov-akvelon**
# Description

This PR moves Android and VS helpers to dedicated files and also changes some variables names in VS installation scripts and helpers to improve readability.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/windows/scripts/build/Install-VSExtensions.ps1
Comment (Alexey-Ayupov) [inline images/windows/scripts/build/Install-VSExtensions.ps1:1]: nitpick: newline at the end

File: images/windows/scripts/helpers/AndroidHelpers.ps1
Comment (Alexey-Ayupov) [inline images/windows/scripts/helpers/AndroidHelpers.ps1:1]: nitpick: newline at the end
What is the format of parameters section in functions? Do we need whitespace between type and variable name? e.g. `[string]$someVar` or `[string] $someVar`

File: images/windows/scripts/helpers/ImageHelpers.psm1
Comment (Alexey-Ayupov) [inline images/windows/scripts/helpers/ImageHelpers.psm1:1]: nitpick: newline at the end

File: images/windows/scripts/helpers/VisualStudioHelpers.ps1
Comment (Alexey-Ayupov) [inline images/windows/scripts/helpers/VisualStudioHelpers.ps1:1]: nitpick: newline at the end

**Review by Alexey-Ayupov — DISMISSED** [review]

**Review by mikhailkoliada — APPROVED** [review]

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**